packages/components/src/utils/index.ts

```diff
@@ -5,3 +5,31 @@ export const uuid = () => {
 
 	return Math.random().toString();
 };
+
+export type ClassNameArg =
+	| string
+	| { [key: string]: boolean | undefined }
+	| undefined;
+export const cls = (...args: ClassNameArg[]) => {
+	let result = '';
+
+	args.forEach((arg, index) => {
+		if (arg) {
+			if (index != 0) {
+				result += ' ';
+			}
+
+			if (typeof arg === 'string') {
+				result += arg;
+			} else {
+				for (let key in arg) {
+					if (arg[key]) {
+						result += key;
```

Comment:
```suggestion
						result += ' ' + key;
```

Wouldn't this result in a missing space?

Reply:
changed it with `.trim()`